core/embed/rust/src/ui/model_tr/layout.rs

```diff
@@ -2054,21 +2054,22 @@ pub static mp_module_trezorui2: Module = obj_module! {
     ///     dry_run: bool,
     ///     info_button: bool,  # unused on TR
     ///     show_info: bool,
-    /// ) -> object:
+    /// ) -> LayoutObj[UiResult]:
     ///    """Device recovery homescreen."""
     Qstr::MP_QSTR_confirm_recovery => obj_fn_kw!(0, new_confirm_recovery).as_obj(),
 
     /// def select_word_count(
     ///     *,
     ///     dry_run: bool,  # unused on TR
     /// ) -> int | str:  # TR returns str
+    /// ) -> LayoutObj[int | str]:
```

Comment:
Seems we should delete the line above to compensate for the new line

Reply:
good catch, and how tf did this not completely break typechecking?

Reply:
b9e1bc8fa4e1ffbe1ce5c7142fd3ec417a86a7f0